build.gradle

```diff
@@ -95,16 +94,31 @@ publishing {
 
     repositories {
         maven {
+            def releasesRepoUrl = 'https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/'
+            def snapshotsRepoUrl = 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+            url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
             name = "OSSRH"
-            url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+            // url = "https://s01.oss.sonatype.org/content/repositories/"
             credentials {
-                username = System.getenv('OSSRH_USERNAME')
-                password = System.getenv('OSSRH_PASSWORD')
+                username = 'VaibhavTechsophy'
```

Comment:
Please take care while adding code before merging. You have now published the password to the world